examples/multisig_plain/lib.rs

```diff
@@ -186,6 +193,17 @@ mod multisig_plain {
         result: Result<(), ()>,
     }
 
+    /// Emitted when a transaction was executed and its output is evaluated.
+    #[ink(event)]
+    struct Evaluation {
+        /// The transaction that was executed.
+        #[ink(topic)]
+        transaction: TransactionId,
+        /// The output of the transaction as bytes.
+        #[ink(topic)]
+        result: Result<Vec<u8>, ()>,
+    }
+
```

Comment:
I think it would be more user friendly to have a single event for this that unifies both events.
E.g. convert `result` type to `Result<Option<Vec<u8>>, ()>` or something like that to signal if there was a return type or not.

Reply:
Done.